.travis.yml

```diff
@@ -19,6 +19,7 @@ before_install:
   - go get github.com/client9/misspell/cmd/misspell
   - go get -u github.com/gordonklaus/ineffassign
   - go get -u github.com/alexkohler/nakedret
+  - go get github.com/lann/builder
```

Comment:
how is this used?  it's seem unrelated to the main change here

Reply:
this is somehow the dependency from `github.com/Masterminds/squirrel` but govendor didn't manage to add this to the main vendor.json file, so I added it here.